apps/angular-store-app/src/app/app.module.ts

```diff
@@ -27,7 +29,11 @@ import { AkitaNgDevtools } from '@datorama/akita-ngdevtools';
     CartModule,
     AuthModule,
     AkitaNgRouterStoreModule,
-    AkitaNgDevtools.forRoot()
+    AkitaNgDevtools.forRoot(),
+    // ERROR in Error during template compile of 'AppModule'
```

Comment:
What do u mean by "An Angular update will fix this"?

Reply:
I tried to run the project with multiple Angular versions for the akita store app.
Angular 8 had thrown an error.
Ivy versions worked fine without issues.
I haven't seen this error before.
Why wouldn't it be possible to use a function call within a decorator?

I'm not sure how to deal with this.
We can update the projects Angular version. That would also take a few changes to the code base.
I tried that already.

Reply:
Ok, I will check it when you finish